runtime/doc/eval.txt

```diff
@@ -4034,6 +4035,19 @@ findfile({name} [, {path} [, {count}]])				*findfile()*
 <		Searches from the directory of the current file upwards until
 		it finds the file "tags.vim".
 
+flatten({list} [, {maxdepth}])					*flatten()*
+		Flatten {list} to depth {maxdepth}.  {maxdepth} means how
```

Comment:
It should be noted that flattening is done in-place, i.e., modifies the list (as other vim list functions)